### ISSUE TITLE
Bad lumina-checkpass installation

### ISSUE THREAD

**Issue by tieugene** [lumina-desktop/lumina#771]
This helps to make rpm with rpm build (using fake root):
```diff
--- src-qt5/core/lumina-checkpass/lumina-checkpass.pro.orig     2021-10-04 01:21:46.239515440 +0300
+++ src-qt5/core/lumina-checkpass/lumina-checkpass.pro  2021-10-04 01:22:26.100834380 +0300
@@ -12,7 +12,7 @@
 
 SOURCES += main.c
 
-perms.path = $$DESTDIR$${PREFIX}/sbin
-perms.extra = "chmod 4555 $$DESTDIR$${PREFIX}/sbin/lumina-checkpass"
+perms.path = $(INSTALL_ROOT)$$DESTDIR$${PREFIX}/sbin
+perms.extra = "chmod 4555 $(INSTALL_ROOT)$$DESTDIR$${PREFIX}/sbin/lumina-checkpass"
 
 INSTALLS += target perms
```

**Comment by q5sys** [comment]
For the time being, do you mind doing that as a patch in rpm src like you've done for https://src.fedoraproject.org/rpms/lumina-desktop/blob/rawhide/f/lumina_icons.patch ?

I'd want to investigate that a little further to make sure it doesn't negatively affect other Distros/OSes before I make that change permanent for everyone. 

When I can verify that it doesn't negatively affect everyone else, I'd be happy to make that change into the master branch.
 
Alternatively, if it'd be helpful to you, I'm fine with maintaining a Fedora specific branch that I can keep in sync with the master branch.  I can includes all the required patches that Fedora needs in that branch, so the fedora source rpms can be cleaner and make the build less effort on downstream packagers.

**Comment by tieugene** [comment]
This was patch for lumina 1.3.0 and an idea was that lumina installes pixmap icons into scalables' dir.
(As Issue #702 not closed the patch is actual yet)

**Comment by q5sys** [comment]
I'll make it a point to address that scalable icon issue, but for this perms issue...  Are you ok with fixing this with a patch in rpm.fedoraproject.org?  Or would you rather me maintain a Fedora specific branch here that I can make those changes in?  Either is an acceptable short term solution to me, I just need to know which you'd prefer.

**Comment by q5sys** [comment]
I'll make a note about this in the release notes, but to give you a heads up.  Checkpass is being disabled in 1.6.2. I mis-remembered its use, and after talking to Ken he let me know that it's part of the unified desktop that's not complete yet.  So I'm disabling it for v1.6.2.  The SVGs have been made and have already been merged into my temp 1.6.2 branch as well.
I still need to get the time to test the perms changes you've asked for to make sure they dont have any issues with other OSes.  That's one thing I've not found the time to do since I need to do a bunch of installs first.

**Comment by q5sys** [comment]
Oh wait... nevermind. I just re-read.  I can close this out, since that perms thing is related to checkpass which is being disabled.
